src/components/Navbar.tsx

```diff
@@ -0,0 +1,90 @@
+import LoginBtn from './LoginBtn';
+import { useTranslation } from 'next-i18next';
+import type { GetStaticProps, InferGetStaticPropsType } from 'next';
+import { getStaticProps } from '@/pages';
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
+
+ //lines below will be needed fo authentication
+// interface _Props {
+//   authData: {
+//     user?: {
+//       name?: string | null | undefined;
+//       email?: string | null | undefined;
+//       image?: string | null | undefined;
+//     };
+//   } | null;
+//   authStatus: 'authenticated' | 'loading' | 'unauthenticated';
+// }
+
+const Navbar = function (
+  _Props: InferGetStaticPropsType<typeof getStaticProps>,
+  //lines below will be needed fo authentication
+  // authData: {
+  //   user?: {
+  //     name?: string | null | undefined;
+  //     email?: string | null | undefined;
+  //     image?: string | null | undefined;
+  //   };
+  // } | null,
+  // authStatus?: 'authenticated' | 'loading' | 'unauthenticated' | 'undefined'
+) {
+  const { t } = useTranslation('navbar');
+
+  return (
+    <div className="navbar bg-[#abd9e1] px-2 shadow-xl">
+      <div className="logo flex-1">
+        <a className="btn-ghost btn text-xl normal-case">
+          <img
+            alt="Journey الرحلة logo"
+            src="https://media1-production-mightynetworks.imgix.net/asset/56053552/JOURNEYUNCROPPED2-28.png?ixlib=rails-4.2.0&amp;auto=format&amp;w=150&amp;h=100&amp;fit=crop&amp;impolicy=Avatar&amp;crop=faces"
+            width="60"
+          />
+        </a>
+      </div>
+      <div className="flex-none">
+        <div className="bell dropdown-end dropdown">
+          <label tabIndex={0} className="bell btn-ghost btn-circle btn">
+            <div className="bell-num indicator">
+              <svg xmlns="http://www.w3.org/2000/svg" xmlnsXlink="http://www.w3.org/1999/xlink" width="25" height="25">
+                <g fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
+                  <path d="M10 5a2 2 0 0 1 4 0a7 7 0 0 1 4 6v3a4 4 0 0 0 2 3H4a4 4 0 0 0 2-3v-3a7 7 0 0 1 4-6"></path>
+                  <path d="M9 17v1a3 3 0 0 0 6 0v-1"></path>
+                </g>
+              </svg>
```

Comment:
❌ 
put the SVG in a separate file in the public folder, under `public/media/icons/[name].svg`

read about best practices to deal with static assets with nextjs

Reply:
Done